### DISCUSSION TITLE
Can you incorporate PnP search filter with out of the box SharePoint Search results page

### DISCUSSION THREAD

**Discussion by amilanc** [microsoft-search/pnp-modern-search#3378] [Q&A] [answered]
SharePoint Out of the box Search results page gives what we need. Is there a way to incorporate PnP Search Filter with Out of the box search results page?

**Comment by kasperbolarsen** [comment, marked as answer]
Hi,

The answer is no, as the filter values are kind of "provided" by the main web part which is the Results web part.

The out of the box filters are pretty rudimentary, so you have a choice to make.
